Formula/polyrhythmix.rb

```diff
@@ -0,0 +1,17 @@
+class Polyrhythmix < Formula
+  desc "Polyrhythmically-inclinded MIDI Drum generator"
+  homepage "https://github.com/dredozubov/polyrhythmix"
+  url "https://github.com/dredozubov/polyrhythmix/archive/refs/tags/0.1.0.tar.gz"
+  sha256 "d4122d34dde52681e217304a8f4c3b10363c964d57f4b2608fb2448b5c83a3c9"
+  license "Unlicense"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/poly", "-K", "4xxxx"
```

Comment:
maybe a better test
```
    expected = <<~EOS
      No output file path was supplied, running a dry run...
      Converges over 1 bar
    EOS
    output = shell_output("#{bin}/poly -K 4xxxx")
    assert_equal expected, output
```

also looks like the version output mismatched with the artifact

```
$ /opt/homebrew/Cellar/polyrhythmix/0.1.0/bin/poly --version
Polyrhythmix 0.1
```

Reply:
Good catch regarding the version, it was hardcoded in this version, I've updated it in the polyrhythmix repo itself and it should be included in the release script later.

Considering the test idea, I'm not sold on that idea, as it increases the potential for breakage with version changes.